app/models/person.rb

```diff
@@ -59,7 +59,7 @@ def next_target
   end
 
   def unconvinced_legislators
-    legislators && legislators.unconvinced
+    legislators && legislators.unconvinced.eligible
   end
```

Comment:
I'm wondering if we can save db calls by making these associations - but not overload the class.